### ISSUE TITLE
Replace the default Sleek favicon

### ISSUE THREAD

**Issue by qerub** [repaircafe/malmo.repaircafe.nu#28]
<img width="285" alt="Screenshot 2019-08-24 at 22 24 17" src="https://user-images.githubusercontent.com/636/63642342-22b6c500-c6be-11e9-825d-fd6e0981c17e.png">

**Comment by qerub** [comment]
See https://github.com/repaircafe/jekyll-sleek/tree/repaircafe/assets/img/icons for the files that need replacements.

**Comment by monsieurhannes** [comment]
Started here #33 
But one question: Do we really have to change it in the template-repo? That would make it quite unflexible... Same thing with the "hero-image", would be good to have that in each "local"-repo. Any suggestions?

**Comment by monsieurhannes** [comment]
Started here #33 
But one question: Do we really have to change it in the template-repo? That would make it quite unflexible... Same thing with the "hero-image", would be good to have that in each "local"-repo. Any suggestions?

**Comment by qerub** [comment]
No. malmo.repaircafe.nu behaves like an overlay for jekyll-sleek so you just have to create files with exactly the same in our repo to override the theme's assets.
